src/components/Pagination/Pagination.tsx

```diff
@@ -32,145 +32,104 @@ export type PaginationProps = {
   showNext?: boolean;
 };
 
-type PageLink = {
-  label: string;
-  ariaLabel: string;
-  pageNumber: number;
-  additionalClassName: string | null;
-  isDisabled?: boolean;
-  isSelected?: boolean;
-  createURL(value: number): string;
-};
-
 const defaultProps = {
```

Comment:
This default props pattern doesn't work well with TypeScript, it lies about some fields being optional, while they can't be. We could default the props line 41 when destructuring.